src/sensors/distance/infrared/analog/InfraredAnalogSensor.hpp

```diff
@@ -6,7 +6,7 @@
  */
 #pragma once
 
-#include <stdint.h> // NOLINT(modernize-deprecated-headers)
+#include <stdint.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'stdint.h'; consider using 'cstdint' instead
```suggestion
#include <cstdint>```